templates/single.html

```diff
@@ -1,47 +1,57 @@
 <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
 
 <!-- wp:group {"tagName":"main"} -->
-<main class="wp-block-group"><!-- wp:group {"layout":{"inherit":true}} -->
-<div class="wp-block-group"><!-- wp:post-title {"level":1,"align":"wide"} /-->
-
-<!-- wp:post-featured-image {"align":"wide"} /-->
-
-<!-- wp:separator {"align":"wide","className":"is-style-wide"} -->
-<hr class="wp-block-separator alignwide is-style-wide"/>
-<!-- /wp:separator --></div>
-<!-- /wp:group -->
-
-<!-- wp:spacer {"height":"var(--wp--preset--spacing--30)"} -->
-<div style="height:var(--wp--preset--spacing--30)" aria-hidden="true" class="wp-block-spacer"></div>
-<!-- /wp:spacer -->
-
-<!-- wp:post-content {"layout":{"inherit":true}} /-->
-
-<!-- wp:spacer {"height":"var(--wp--preset--spacing--30)"} -->
-<div style="height:var(--wp--preset--spacing--30)" aria-hidden="true" class="wp-block-spacer"></div>
-<!-- /wp:spacer -->
-
-<!-- wp:group {"layout":{"inherit":true}} -->
-<div class="wp-block-group"><!-- wp:group {"layout":{"type":"flex"}} -->
-<div class="wp-block-group"><!-- wp:post-date {"format":"F j, Y"} /-->
-
-<!-- wp:post-author {"showAvatar":false} /-->
-
-<!-- wp:post-terms {"term":"category"} /-->
-
-<!-- wp:post-terms {"term":"post_tag"} /--></div>
-<!-- /wp:group -->
-
-<!-- wp:spacer {"height":"var(--wp--preset--spacing--30)"} -->
-<div style="height:var(--wp--preset--spacing--30)" aria-hidden="true" class="wp-block-spacer"></div>
-<!-- /wp:spacer -->
-
-<!-- wp:separator {"className":"is-style-wide"} -->
-<hr class="wp-block-separator is-style-wide"/>
-<!-- /wp:separator -->
-
-<!-- wp:post-comments /--></div>
-<!-- /wp:group --></main>
+<main class="wp-block-group">
+	<!-- wp:group {"layout":{"inherit":true}} -->
+	<div class="wp-block-group">
+		<!-- wp:post-featured-image {"align":"wide"} /-->
+		<!-- wp:post-title {"level":1,"align":"wide","style":{"spacing":{"margin":{"top":"7rem","bottom":"7rem"}}}} /-->
+	</div>
+	<!-- /wp:group -->
+
+	<!-- wp:post-content {"layout":{"inherit":true}} /-->
+
+	<!-- wp:group {"style":{"spacing":{"margin":{"top":"7rem","bottom":"7rem"}}},"layout":{"inherit":true}} -->
+	<div class="wp-block-group" style="margin-top:7rem;margin-bottom:7rem">
+		<!-- wp:separator {"opacity":"css","align":"wide","className":"is-style-wide"} -->
+		<hr class="wp-block-separator alignwide has-css-opacity is-style-wide"/>
+		<!-- /wp:separator -->
+
+		<!-- wp:columns {"align":"wide","style":{"spacing":{"margin":{"top":"var(--wp--preset--spacing--70)"},"blockGap":"var(--wp--preset--spacing--70)"}},"fontSize":"small"} -->
```

Comment:
The paragraph block, post date and terms has different font sizes, adding the font size to the columns evens this out.
The caveat is that this can't be changed with style variations.